db/seeds.rb

```diff
@@ -5,3 +5,57 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+# apartments = [
+#     {
+#     user_id: 1
+#     street: '444 B St.',
+#     city: 'Developer',
+#     state: 'CA',
+#     manager_name: 'Luis',
+#     manager_contact_email: 'luis@gmail.com'
+#     manager_rental_price: 3321.90,
+#     number_of_bedrooms: 4
+#     number_of_bathrooms: 2,
+#     allow_pets: false
+#     },
+#     user_id: 1,
+#     street: "3742 Hamill Avenue",
+#     city: "San Diego",
+#     state: "CA",
+#     manager_name: "Kira Beasley",
+#     manager_email: "kbeasley@123.com",
+#     monthly_rent: 1850,
+#     number_of_bedrooms: 2,
+#     number_of_bathrooms: 1.5,
+#     allow_pets: true,
+#   },
+#   {
+#     user_id: 1,
+#     street: "814 Carriage Court",
+#     city: "San Diego",
+#     state: "CA",
+#     manager_name: "Shannon Atkinson",
+#     manager_email: "satkinson@123.com",
+#     monthly_rent: 2230,
+#     number_of_bedrooms: 3,
+#     number_of_bathrooms: 2,
+#     allow_pets: true,
+#   },
+#   {
+#     user_id: 1,
+#     street: "3956 Poplar Avenue",
+#     city: "San Diego",
+#     state: "CA",
+#     manager_name: "Edith Farrell",
+#     manager_email: "efarrell@123.com",
+#     monthly_rent: 1575,
+#     number_of_bedrooms: 1,
+#     number_of_bathrooms: 1.5,
+#     allow_pets: false,
+#   }
+# ]
+
+# apartments.each do |apartment|
+#     Apartment.creat apartment
```

Comment:
Looks like you are on the right track here. But you may want to consider the assumption you are making about the existence of a user with the id of 1.